ui/app/adapters/generated-item-list.js

```diff
@@ -28,8 +29,10 @@ export default class GeneratedItemListAdapter extends ApplicationAdapter {
     return this.paths.deletePath || '';
   }
 
-  getDynamicApiPath(id) {
-    const result = this.store.peekRecord('auth-method', id);
+  getDynamicApiPath() {
+    const result = getOwner(this)
+      .lookup('route:vault.cluster.access.method')
+      .modelFor('vault.cluster.access.method');
```

Comment:
There are no longer `auth-method` models loaded in the store so for now I updated this to grab the model which is the auth method resource from the method route. This `generated-item-list` pattern will be going away so this is a short term solution until the next phase of the auth method migration.